tests/Makefile

```diff
@@ -25,8 +25,9 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 
-# distutils warnings are caused by an old version of virtualenv in travis
-export PYTHONWARNINGS = error,ignore::DeprecationWarning:distutils
+# Do not fail on DeprecationWarning caused by virtualenv, which might come from
+# the distutils or site modules.
+export PYTHONWARNINGS = error,ignore::DeprecationWarning:distutils,ignore::DeprecationWarning:site
```

Comment:
To be clear: I'm fine either with this change or with removing the existing filter.